### ISSUE TITLE
Online Dashboard Script

### ISSUE THREAD

**Issue by Incrested** [Miku-Chan-Devs/Miku-Chan#46]
I was thinking we could do PHP or (perhaps) Node.js for the server. @ultraMLGcode what do you reckon it should be going for?

**Comment by Ultra03** [comment]
PHP should be what we're going for as it's the best form of communication between the website and a MySQL database.

**Comment by Ultra03** [comment]
Going back on what I said a while ago, we could actually use JS/jQuery and use AJAX calls to the server. May/may not be more difficult to achieve, I'm not sure atm.

**Comment by Incrested** [comment]
Yeah, I'm gonna start on new front-end after my workout today and then see about getting a decent front-end to work all this. Not sure on database though: Heroku + PHP + MySQL has had some bad mixings. Possibly a "database" using something ***super hard core*** like JSON or XML (I kid, obviously. Tho it is important to look into)

**Comment by Ultra03** [comment]
Yeah; here’s a list of languages we could use for the backend (and backend/frontend communication):

- Node.js (obviously)
- PHP
- JS/jQuery with AJAX
- Python using [Flask](http://flask.pocoo.org)

Those are just off the top of my head. There’s definitely more. Post here if you have any suggestions.

**Comment by Incrested** [comment]
Yeah, I think Node and jQuery + AJAX is our easiest solution while still being productive. I'll move the dashboard to a later date and start prepping a project board so all of this can finally be organized.
